content/docs/09.administrator-guide/upgrades.md

```diff
@@ -40,7 +40,9 @@ Next to all bug fixes and enhancements, you can find a dedicated section called
 
 If running Kestra in separate components you should:
 - Stop the executors and the scheduler
-- Stop the workers - there is a graceful shutdown (which can be configured but I'm not sure we already document how to configure this) and automatic task resubmission.
+- Stop the workers - there is a graceful shutdown period to finish active jobs before closing the worker. This is set by default to be: `kestra.server.terminateGracePeriod = '5m'` but is configurable.
```

Comment:
Maybe you could add a link to the configuration reference when you say it's configurable?